routes/route.py

```diff
@@ -15,12 +16,19 @@ def resource_not_found(e):
     return jsonify(error=str(e)), 400
 
 
-@routes.route('/test', methods=['GET'])
+@routes.route('/CompareSequence/', methods=['POST'])
```

Comment:
I'm not sure if it's good practice to have capital letters in the url path... I'd have to check documentation though

Reply:
sounds good

Reply:
ok so if you check [here](https://restfulapi.net/resource-naming/) it says to avoid capital letters when possible, and a bunch of other conventions

![image](https://user-images.githubusercontent.com/35381483/110276159-8d79d980-7fa0-11eb-936d-6835062eabec.png)

so you could name the path `/compare-sequence` instead 

also ditch the trailing forward slash

![image](https://user-images.githubusercontent.com/35381483/110276281-cc0f9400-7fa0-11eb-8adf-b864d795340f.png)